src/runtime/ArrayBufferObject.h

```diff
@@ -98,6 +110,43 @@ class ArrayBufferObject : public Object {
         return true;
     }
 
+    Value getTypedValueFromBuffer(ExecutionState& state, unsigned byteindex, TypedArrayType type)
+    {
+        Value value;
+        switch (type) {
+        case TypedArrayType::Int8:
+            value = getValueFromBuffer<int8_t>(state, byteindex, true);
```

Comment:
I think you can directly return in all cases.

Reply:
That's a nice catch, that flew past by me. I updated the PR with this change.

Reply:
I think this code path has not been updated.